examples/optimize_evoked.py

```diff
@@ -0,0 +1,82 @@
+"""
+=================================================
+08. Optimize simulated evoked response parameters
+=================================================
+
+This example demonstrates how to optimize the parameters
+of the model simulation to match an experimental dipole waveform.
+"""
+
+# Authors: Blake Caldwell <blake_caldwell@brown.edu>
+#          Mainak Jas <mjas@mgh.harvard.edu>
+
+import os.path as op
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+###############################################################################
+# Let us import hnn_core
+
+import hnn_core
+from hnn_core.dipole import simulate_dipole
+from hnn_core import read_params, Dipole, MPIBackend, jones_2009_model
+
+hnn_core_root = op.join(op.dirname(hnn_core.__file__))
+
+# The number of cores may need modifying depending on your current machine.
+n_procs = 10
+
+###############################################################################
+# First, we will load experimental data into Dipole object.
+#
+# This is a different experiment than the one to which the base parameters were
+# tuned. So, the initial RMSE will be large, giving the optimization procedure
+# a lot to work with.
+from urllib.request import urlretrieve
+
+data_url = ('https://raw.githubusercontent.com/jonescompneurolab/hnn/master/'
+            'data/MEG_detection_data/S1_SupraT.txt')
+urlretrieve(data_url, 'S1_SupraT.txt')
+extdata = np.loadtxt('S1_SupraT.txt')
+exp_dpl = Dipole(extdata[:, 0], np.c_[extdata[:, 1],
+                 extdata[:, 1], extdata[:, 1]])
+
+###############################################################################
+# Read the base parameters from a file
+params_fname = op.join(hnn_core_root, 'param', 'default.json')
+params = read_params(params_fname)
+
+###############################################################################
+# Let's first simulate the dipole with the initial parameters.
+net = jones_2009_model(params, add_drives_from_params=True)
+with MPIBackend(n_procs=n_procs):
+    initial_dpl = simulate_dipole(net, n_trials=1)
```

Comment:
```suggestion
    initial_dpl = simulate_dipole(net, n_trials=1, postproc=True)
```
Sneaky bugger :wink

Reply:
Don't actually commit this, I'm just pointing out a line that can fix the initial comparison. Scaling still needs to be fixed inside `optimization.py1

Reply:
indeed, this is what I was suspecting. Let me give this a shot. Need to get it to work first ... after that we can think of the optimal API :)